package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "build": "tsc -p .",
     "lint": "tslint -p .",
-    "test": "jest --ci --forceExit"
+    "test": "jest --ci"
```

Comment:
this will still complain about potentially open handles because jest doesn't quit within 1 second of terminating, but this is a false positive ATM